core/src/main/java/com/opensymphony/xwork2/interceptor/ValidationAware.java

```diff
@@ -26,7 +26,7 @@
  * ValidationAware classes can accept Action (class level) or field level error messages. Action level messages are kept
  * in a Collection. Field level error messages are kept in a Map from String field name to a List of field error msgs.
  *
- * @author plightbo 
+ * @author plightbo
```

Comment:
You can remove the `@author` tag next time (recommended by ASF)

Reply:
Done :)